sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -737,6 +737,15 @@ def _add_argparse_args(cls, parser):
               '"<process to execute>", "env":{"<Environment variables 1>": '
               '"<ENV_VAL>"} }. All fields in the json are optional except '
               'command.'))
+    parser.add_argument(
+        '--sdk-worker-parallelism', default=None,
+        help=('Sets the number of sdk worker processes that will run on each '
+              'worker node. Default is 1. If 0, it will be automatically set '
```

Comment:
I'd leave 0 a bit unspecified, e.g. it will be chose by the runner (e.g. by looking at the number of CPU cores on the worker machine).

Reply:
@angoenka Why does helpstring say "Default is 1" but in parser instruction we have `default=None`?

Reply:
I kept this null to match the behavior in java https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/options/PortablePipelineOptions.java#L74